src/js/program/media-controller.js

```diff
@@ -101,17 +101,10 @@ export default class MediaController extends Eventable {
     }
 
     detach() {
-        const { item, mediaModel, provider } = this;
+        const { provider } = this;
         this.thenPlayPromise.cancel();
         provider.detachMedia();
         this.attached = false;
-
-        // If detaching to play a midroll (pos > 0), ensure that the player resumes at the detached time by setting starttime
-        // We don't need to do this for prerolls because the player re-attaches at time 0 by default
-        const pos = mediaModel.get('position');
```

Comment:
detach gets called in [`set background`](https://github.com/jwplayer/jwplayer/blob/72b1b9ced8ecf6124e37ed2a55097cff7cb208df/src/js/program/media-controller.js#L252). Is it ok if `item.starttime = pos;` is not being called when an item is being backgrounded ?

Reply:
Yes. because backgrounded items are paused and do not need `starttime` updated to resume at the correct position.